index.md

```diff
@@ -14,17 +14,18 @@ attendees a strong foundation in the fundamentals of R, and to teach best
 practices for scientific computing: breaking down analyses into modular units,
 task automation, and encapsulation.
 
-Note that this workshop will focus on teaching the fundamentals of the 
+Note that this workshop will focus on teaching the fundamentals of the
 programming language R, and will not teach statistical analysis.
 
 A variety of third party packages are used throughout this workshop. These
-are not necessarily the best, nor are they comprehensive, but they are 
-packages we find useful, and have been chosen primarily for their 
+are not necessarily the best, nor are they comprehensive, but they are
+packages we find useful, and have been chosen primarily for their
 usability.
 
 > ## Prerequisites {.prereq}
 >
-> Have attended the Shell and Git sessions.
+> Understand that computers store data and instructions (programs, scripts etc.) in files. Files are organised in directories (folders).
+> Know how to access files not in the working directory by specifying the `path`.
```

Comment:
Not sure 'path' should be styled this way, but I can see you want to highlight it (maybe *emphasize*?).
I like your wording but maybe I would keep it at "Understand the filesystem and know how to navigate it from the command line."